deploy/docker/templates/appsmith_starting.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" http-equiv="refresh" content="3"/>
+    <link
+      rel="shortcut icon"
+      href="https://res.cloudinary.com/dwpfockn8/image/upload/v1597920848/favicons/favicon-orange_pxfmdc.ico"
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Appsmith</title>
+  </head>
+  <body>
+    <div class="main">
+      <p class="bold-text">
+        Appsmith is Starting.
```

Comment:
Nit: Please use sentence casing in all user visible text. 
```suggestion
        Appsmith is starting.
```